### PR TITLE
update external links

### PR DESCRIPTION
**Pull request by jamescott**

### CODE REVIEW

**Review by Jacksonbaker323 — APPROVED** [review]
lgtm

File: amperity_datagrid/source/bridge_snowflake.rst [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+          :start-after: .. credential-snowflake-region-start
+          :end-before: .. credential-snowflake-region-end
+
+
+   * - .. image:: ../../images/steps-arrow-off-black.png
+          :width: 60 px
+          :alt: Requirement 3.
+          :align: left
+          :class: no-scaled-link
+     - Before you can configure Amperity Bridge for data sharing with Snowflake you must :ref:`create a share in Snowflake <bridge-snowflake-sync-amperity-configure-snowflake-create-share>`, add a secure share identifier to that share, identify the correct account locator, and then add the :ref:`Amperity account locator ID <bridge-snowflake-sync-amperity-configure-snowflake-account-locator>` to the share.
```
Comment (lukeneil) [inline amperity_datagrid/source/bridge_snowflake.rst:105]: Before you can configure Amperity Bridge for data sharing with Snowflake you mustcreate a share in Snowflake, add a secure share identifier to that share, identify the region for your Snowflake account, and add the locator ID for the Amperity account in your region as a consumer of the share.

Use the [CURRENT_REGION](https://docs.snowflake.com/en/sql-reference/functions/current_region) function to return the name of the region for the account where the current user is logged in.

File: amperity_datagrid/source/bridge_snowflake.rst [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+
+       Use the `CURRENT_ACCOUNT <https://docs.snowflake.com/en/sql-reference/functions/current_account>`__ |ext_link| argument in |destination-name| to return the locator ID for your Snowflake account.
+
+       .. important:: Secure data sharing should :ref:`only share secure views <bridge-snowflake-sync-amperity-configure-snowflake-secure-views>` with Amperity.
```
Comment (lukeneil) [inline amperity_datagrid/source/bridge_snowflake.rst:109]: This is a bit ambigous and sounds like they should _only_ share secure views which isn't true - in reality they can share other tables and secure objects with Amperity - but we want to discourage sharing _non-secured views_ specifically.

I would say something like:

When sharing views via secure data sharing, Snowflake strongly recommends only sharing secure views. While it is possible to share data in [non-secured views](https://docs.snowflake.com/en/user-guide/data-sharing-views#label-sharing-non-secure-view-limitations), this is not a reccommended practice.

File: amperity_datagrid/source/bridge_snowflake.rst [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+#. For the share, select a source database, along with one or more target objects within the source database.
+#. A **Secure Share Identifier** is required for data sharing with Amperity.
+
+   A secure share identifier in Snowflake is a unique string---A-Z, 0-9, $, and _ (underscores)---that is added when creating a share. This string must be configured in Amperity to enable the bridge.
+
+#. Find the :ref:`Amperity account locator ID <bridge-snowflake-sync-amperity-configure-snowflake-account-locator>`, and then add it to the configuration for the **Secure Share Identifier**.
+
+#. Copy the secure share identifier from the Snowflake user interface, and then save the share.
+
+#. Click **Add consumer**, and then configure the share to use the :ref:`Amperity account locator ID <bridge-snowflake-sync-amperity-configure-snowflake-account-locator>` as the consumer.
```
Comment (lukeneil) [inline amperity_datagrid/source/bridge_snowflake.rst:171]: Amperity account locator ID _in the same region_ as the consumer

File: amperity_datagrid/source/bridge_snowflake.rst [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+
+
+.. _bridge-snowflake-sync-amperity-configure-snowflake:
+
+Configure Snowflake
+--------------------------------------------------
+
+.. bridge-snowflake-sync-amperity-configure-snowflake-start
+
+Before you can configure Amperity Bridge for data sharing with Snowflake you must create a share in Snowflake, add a secure share identifier to that share, and then identify the correct account locator.
```
Comment (lukeneil) [inline amperity_datagrid/source/bridge_snowflake.rst:139]: identify _and add_ the correct Amperity account locator as a consumer for the share

**Pull request merged**